### DISCUSSION TITLE
Profile resolution spec and adding a control title

### DISCUSSION THREAD

**Discussion by galtm** [usnistgov/OSCAL#1306] [Q&A] [answered]
In a profile's alter object, the add object can contain a title.  The catalog schema permits only one title in a control. Is it correct to assume that the profile resolver should *replace* the control's original title with the new one instead of adding it? More generally, if the profile has multiple alter/add/title patterns that target the same control, should the profile resolver keep only the last title?

It would be great if the spec mentioned the alter/add/title pattern to indicate the behavior explicitly. Thanks!

**Comment by david-waltermire** [comment, marked as answer]
@galtm Yes. I believe your interpretation is correct. Would you post a PR addressing this ambiguity in the specification?

**Reply by galtm** [reply]
@david-waltermire-nist , Sure. On which branch should I base my change?

**Reply by galtm** [reply]
@david-waltermire-nist , I created #1320 to clarify this matter in the specification.

Cc: @wendellpiez
